link/src/main/java/com/stripe/android/link/ui/inline/LinkInlineSignup.kt

```diff
@@ -103,6 +109,7 @@ fun LinkInlineSignup(
     }
 }
 
+@OptIn(ExperimentalFoundationApi::class)
```

Comment:
Is there any other way?

Reply:
Unfortunately there is nothing stable in Jetpack Compose that does something similar.